specifications.yml

```diff
@@ -641,6 +641,10 @@ components:
           description: Date invoice was sent to customer
           format: date
           example: "2022-04-07"
+        status:
+          type: string
+          description: Invoice status
+          example: "waiting-for-payment"
```

Comment:
If we want to do it as on Moank, i'd document is an an enum of "paid", "late", "pending"
I think the account status that we use right now is returning paid or late already but to double check

Reply:
The account status right now is returning order statuses which can vary quite a bit, i think erik is working on adding invoice statuses which will actually make it that enum